packages/ember-runtime/tests/suites/enumerable/includes.js

```diff
@@ -0,0 +1,28 @@
+import {SuiteModuleBuilder} from 'ember-runtime/tests/suites/suite';
+
+var suite = SuiteModuleBuilder.create();
+
+suite.module('includes');
+
+suite.test('includes returns true if items is in enumerable', function() {
```

Comment:
Once the method itself is behind a feature flag, you will also need to wrap these tests inside an `isEnabled` check.